README.md

```diff
@@ -344,6 +344,30 @@ a daily ubuntu-advantage-tools image.
 UACLIENT_BEHAVE_REUSE_IMAGE=your-custom-image-id tox -e behave-awspro-20.04
 ```
 
+### MOTD Messages
+
+Since ubuntu-advantage-tools is responsible for enabling ESM services, we advertise them on different
+applications thorough the system, such as MOTD and apt commands like upgrade.
+
+To verify that the MOTD message is advertising the ESM packages, ensure that we have ESM source list
+files in the system. If that is the case, please run the following commands to update the state of
+MOTD and display the message:
+
+```sh
+# Make sure ubuntu-advantage-tools version >= 27.0
+ua version
+# Make apt aware of the ESM source files
+sudo apt update
+# This tool allows us to more easily update the MOTD message in the sytem
+sudo apt install update-motd
+# Generates ubuntu-advantage-tools messages that should be delivered to MOTD
+python3 /usr/lib/ubuntu-advantage/ua_update_messaging.py
```

Comment:
Idea: we could recommend running the systemd service instead of directly running the python script? 
```suggestion
systemctl start ua-messaging.service
```

Related: We could also mention in the paragraph above that normally these messages get updated with a systemd timer periodically, which is why they might not show up immediately.